kod/object/passive/spell/fade.kod

```diff
@@ -69,12 +69,12 @@ messages:
 
    CanPayCosts(who=$,lTargets=$)
    {
-      % check for enchantment already applied
+      local oIvisibilitySpell;
+      % Remove existing ivisibility spell if it exists
       if Send(who,@CheckPlayerFlag,#flag=PFLAG_INVISIBLE)
       {
-         Send(who,@MsgSendUser,#message_rsc=fade_already_enchanted);
-         
-         return FALSE;
+         oIvisibilitySpell = Send(SYS,@FindSpellByNum,#num=SID_INVISIBILITY);
```

Comment:
It looks like PFLAG_INVISIBLE does not imply the invisibility spell.  For example, Fade itself sets this player flag.  Is it a good idea to attempt to remove the enchantment if it isn't applied?  With more research, it looks like it might be safe, but I'm not 100% sure.